feat(content): Ka'het: Voice From The Past (#11264)
endless-sky/endless-sky: data/kahet/kahet missions.txt

	"reputation: Ka'sei" = 1

event "patir mystery timer"

ship "Builder Satellite"
	sprite "ship/builder satellite"
	attributes
		category "Drone"

> 		hull 150
> 		mass 10
> 		drag 2

Reviewer: ```suggestion
		"hull" 150
		"mass" 10
		"drag" 2
```

		"heat dissipation" 1

ship "Builder Satellite" "Builder Satellite 2"
	explode "tiny explosion" 5
	explode "small explosion" 2

mission "Ka'het: Voice From The Past 0"

> 	invisible
> 	landing

Reviewer: ```suggestion
	landing
	invisible
```

	to offer
		has "event: remnant: cognizance calm"
	npc assist save
		government "Builder"
		system "Fereti"

> 		personality derelict waiting target

Reviewer: ```suggestion
		personality derelict target waiting
```

			clear "builder signal"

mission "Ka'het: Voice From The Past 1"
	name "Builder signal"
	description "Look for Gavriil aboard <destination> to ask him about the Builder satellite."
	source
		government "Remnant"
		attributes "remnant primary"
	destination "Ssil Vida"
	to offer
		has "builder signal"
	on offer
		conversation
			`Do you want to ask the Remnant about the communication received from the Builder satellite?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
			`	You spend at least half an hour looking for someone in the spaceport, but you can't seem to find anyone familiar. Just as you're starting to lose hope, you notice a flurry of movement in the direction of one of the scientific laboratories.`
			`	Moving closer, you notice the crowd moving away to leave space for a convoy of heavy machinery to pass through. Though you can't seem to identify their function, you do recognize the person leading them: it's Plume, the scientist leading the Void Sprite research program.`
			choice
				`	(Try to catch his attention.)`

> 			`	You wave your arm to try to catch his attention, slowly squeezing through the crowd. "Captain <first>! I did not expect to see you here. You were part of the fleet as well?" he signs, gesturing toward the cargo being transported with excitement.`

Reviewer: It's a little ambiguous who's squeezing through the crowd here. Could that be clarified?

Author: Changed to "You wave your arm to try to catch his attention as you slowly squeeze through the crowd"

			`	Seeing your confused expression, he understands his mistake. "Sorry, I have been having spending some long days in the laboratory recently," he continues with a slight embarrassment. "It is good that we have such an abundance of data, mind you, but the preparations for this expedition have been exhausting."`
			choice
				`	"What are you transporting?"`
				`	"I have found something interesting in Fereti."`
					goto satellite

> 			`	"Cryogenic vats, full of Void Sprite material. All the work done in Esquiline is finally bearing its fruits, and this at least the more noticeable result. It did not take long after we left them roam in Pantica that we started to realize their close passes to the station left visible traces of diaphoresis all over it. All we had to do was put detachable panels over its surface, so that we could start gathering and analyzing those samples.`

Reviewer: ```suggestion
			`	"Cryogenic vats, full of Void Sprite material. All the work done in Esquiline is finally bearing its fruits, and this is the more noticeable result. It did not take long after we left them to roam in Pantica that we started to realize their close passes to the station left visible traces of diaphoresis all over it. All we had to do was put detachable panels over its surface, so that we could start gathering and analyzing those samples.`
```

			choice
				`	"Who should I ask then?"`
				`	"That's a shame."`
			`	Plume stops to oversee a group of technicians unloading the content of the first vehicle, gesturing you to wait for him. He comes back to you a few minutes later. "I have had an idea. Follow me."`
			`	"Here," he signs after a short walk, gesturing toward a wall-mounted terminal. "I may not have enough engineering experience to help you directly, but if there is a place where that information might be, it is this." He grabs your hand and passes it below a scanner before continuing.`
			`	"This pad contains the extracts of all our published engineering papers - the publicly available ones, at least. If what you have encountered has been studied previously, it should be here somewhere." Plume returns to his job after showing you the basics, signing to give him a call if you were to need more help.`
			`	Left by yourself, you grab a nearby seat and start making some searches inside the terminal. The first half hour does not return any useful result: there are millions of papers to choose from, and the initial queries you come up with find either thousands of possible matches, or none at all.`

> 			`	Eventually, however, you start getting the hand of it. You find one paper that describes the phenomena you observed, and another, and more. By the end of it, you have ended up with and saved at least a dozen relevant papers.`

Reviewer: ```suggestion
			`	Eventually, however, you start getting the hang of it. You find one paper that describes the phenomena you observed, and another, and more. By the end of it, you have ended up with and saved at least a dozen relevant papers.`
```

			`	Though you can't understand any of them in depth, a detail catches your eye: three of those papers were authored by Gavriil, the engineer that you last left in <system>. If there's someone who could be able to get you to the end of this, it has to be him.`
			`	You wave goodbye to Plume, still busy transferring the delicate cargo to the safety of the laboratory, and climb aboard your ship. Next stop: <planet>.`
				accept
	on accept
		fail "Ka'het: Voice From The Past 0"
	on complete
		conversation

> 			`The more time passes, and the more Ssil Vida starts to look like a Remnant outpost. The local spaceport, except for the sheer size of each room, has followed a transformation similar to Esquiline, with walls covered by mazes of piping and dozens of active work stations expanding the already operational outfitter. There is no need to wander around, however: Gavriil is waiting for you right beside the landing pad.`

Reviewer: ```suggestion
			`The more time passes, the more Ssil Vida starts to look like a Remnant outpost. The local spaceport, except for the sheer size of each room, has followed a transformation similar to Esquiline, with walls covered by mazes of piping and dozens of active work stations expanding the already operational outfitter. There is no need to wander around, however: Gavriil is waiting for you right beside the landing pad.`
```

			`	"Happy to see you again," he gestures warmly. "I hope the Embers blessed your journey."`
			choice
				`	"They did."`
				`	(Tell him about the encounter.)`

> 			`	You take him aside and start explaining again your encounter with the Builder satellite, but he stops you. "Plume has told me everything a few days ago. It is always delightful to see something one has theorized about become reality," he continues proudly. "What you have observed is something usually called 'zombie satellite'. Not too technical of an expression maybe, but certainly effective.`

Reviewer: ```suggestion
			`	You take him aside and start explaining again your encounter with the Builder satellite, but he stops you. "Plume told me everything a few days ago. It is always delightful to see something one has theorized about become reality," he continues proudly. "What you have observed is something usually called 'zombie satellite'. Not too technical of an expression maybe, but certainly effective.`
```

			choice
				`	"How did it come back to life?"`
				`	"Can you take a look at the signal it sent?"`
					goto signal
			`	"I have a few ideas, but there is no way to be sure. One of the most frequent causes is the software successfully rebooting after a memory failure, practically resetting it to the factory setup, but without more information this is at best a guess on my part."`
			label signal
			`	You hand him a data chip with the communication you received from the satellite, which he quickly puts into his pocket. "I will try to take a few hours off to give it a first pass with the instruments we have here. While they are not the best suited for this sort of analysis, I remember a few tricks to try to get something usable out of it," he concludes.`
			`	"In the meanwhile, feel free to take a look at the changes we have been making here. I will give you a call from the spaceport when I have something to show."`

mission "Ka'het: Voice From The Past 2"
	name "Outpost in Kumkapi"
	description "Bring Gavriil to <destination> to prepare for the capture the Builder satellite."
	source "Ssil Vida"
	destination "Kumkapi"

> 	passengers 1

Reviewer: ```suggestion
	passengers 1
	deadline
```
Based on line 874, Gavriil's only got a few days.

	to offer
		has "Ka'het: Voice From The Past 1: done"
	on offer
		conversation
			`Ssil Vida's outfitter has been built atop the original storage platforms of the station, far too large for the Remnant's logistics system. While it does not seem to be particularly well equipped, it's still quite impressive that they were able to set it up on a completely alien infrastructure so quickly.`
			`	Your walk comes to an end when Gavriil's name appears on your communicator. You turn around and stride back to the spaceport, where you find him waiting at a table.`
			`	"I have good news and bad news," he signs with a slight frown. "How much do you know about the history of our exploration of the Graveyard?"`
			choice
				`	"Not much."`
				`	"Enough."`
					goto waaaah

> 			`	"After the loss a number of vehicles to what we now know to be the Ka'het, our ancestors developed and adopted new standards to reduce the risk of such expeditions into the depth of the Ember Wastes. Cloaks were to be kept active at all times while inside the Graveyard, limiting the range of our archeological missions to the ever-increasing capacity of our fuel tanks.`

Reviewer: ```suggestion
			`	"After the loss of a number of vehicles to what we now know to be the Ka'het, our ancestors developed and adopted new standards to reduce the risk of such expeditions into the depth of the Ember Wastes. Cloaks were to be kept active at all times while inside the Graveyard, limiting the range of our archeological missions to the ever-increasing capacity of our fuel tanks.`
```

			`	"Despite making it so most expeditions would only reach the northern systems, it is these guidelines that led to the golden age of Builder archeology, giving us incredible insights into the history of this region and of its inhabitants. The culmination of this effort was the partial translation of their written language, as well as the decoding of some of their programming. The problem, however, is that this did not extend to the spoken one.`
				goto continue
			label waaaah
			`	He looks amused. "I will try to keep things short then. During our archeological effort, we were able to translate most of the written Builder language and decode some of their programming. The problem, however, is that this did not extend to their spoken language.`
			label continue

> 			`	What we have discovered in the analysis is that the communication you received contains some kind of audio recording, the meaning of which remains a mystery. I have transmitted it to Aventine so that xenolinguists will be able to archive it for further studies, but the purpose of this signal, and consequently of the satellite, will remain unknown."`

Reviewer: ```suggestion
			`	"What we have discovered in the analysis is that the communication you received contains some kind of audio recording, the meaning of which remains a mystery. I have transmitted it to Aventine so that xenolinguists will be able to archive it for further studies, but the purpose of this signal, and consequently of the satellite, will remain unknown."`
```

			`	"On the other hand, we have also been completing milestones ahead of schedule lately, and there are plenty of experienced people here who could take over for some time. I can afford to leave the reigns to someone else for a few days.`
			`	"Plus," he concludes with a smile, "it will be an opportunity to see some faces I have not met in a few years, and reminisce about the past for a bit. I should have been doing this for a while now."`
				accept

mission "Ka'het: Voice From The Past 3"
	landing
	name "Builder satellite"
	description "Capture the Builder satellite in <waypoints> and bring it to <destination>."
	source "Kumkapi"
	waypoint Fereti
	cargo "Satellite Capture Equipment" 45
	to offer
		has "Ka'het: Voice From The Past 2: done"
	on offer
		conversation
			`The descent toward the landing site is slow, and quite gradual. Dozens of mining outposts are scattered over the surface, some of which quite big, but none of them gives the impression of being in active use. Even the landing pads, almost all empty, have started show a thin layer of dust and regolith.`

> 			`	"Just a few decades ago, these buildings were sprawling of activity," Gavriil explains, keeping his eyes on the horizon. "Everyone was on the hunt for Key Stone veins outside of Viminal, and planetary systems tend to have similar characteristics between each other. I will spare you the details, but this turned out not to be the case with <planet>, and most of the infrastructure that had built up here was left without a clear purpose."`

Reviewer: ```suggestion
			`	"Just a few decades ago, these buildings were sprawling with activity," Gavriil explains, keeping his eyes on the horizon. "Everyone was on the hunt for Key Stone veins outside of Viminal, and planetary systems tend to have similar characteristics between each other. I will spare you the details, but this turned out not to be the case with <planet>, and most of the infrastructure that had built up here was left without a clear purpose."`
```

			`	Following the landing trajectory, you steer left after passing over a canyon and prepare for the final approach. You notice his eyes light up when the target outpost comes into view, just for an instant. "So as not to leave it to rot, a few projects to reuse the abandoned facilities were given the go ahead. The one we are going to visit today has become a major testing facility for advanced technologies, may those be built by us or recovered from other sources."`
			choice
				`	"I see."`
					goto skip
				`	"Was the Key Stone search successful?"`
			`	"Not as much as we were hoping. The only other place to turn a positive result was Eavine, a gas giant's moon located right in the middle of Ka'het-controlled space. It was a scientifically interesting finding, but not one that we could make use of in the near future."`
			label skip

> 			`	By the time you complete the post-landing checklist, Gavriil has already left the ship your ship to meet up with the people on the ground. The crowd that has been gathering around the pad give him a warm welcome, to which he replies cheerfully. He must have been spending a lot of time on this moon when he was younger.`

Reviewer: ```suggestion
			`	By the time you complete the post-landing checklist, Gavriil has already left the ship your ship to meet up with the people on the ground. The crowd that has been gathering around the pad give him a warm welcome, to which he replies cheerfully. He must have spent a lot of time on this moon when he was younger.`
```

			`	"Connecting with the spacecraft after you carry it here is going to be a bit more complicated. We have the machines necessary to keep it stored, but I had to order some special devices from Aventine to download its data after all is said and done. Either way, a fleet will have brought it here before you are back, so it should not pose a problem."`
			`	You help them load the satellite capture equipment in your ship, and make sure to leave enough space for the spacecraft itself when it will be time. "I will stay here to help them prepare for the satellite," Gavriil signs, "and hopefully remain updated with Ssil Vida on the progress made in my absence. See you in a couple weeks."`
				accept
	npc assist
		government "Builder"
		system "Fereti"
		personality derelict waiting target
		ship "Builder Satellite" "Valii'sei"
		conversation

> 			`You approach the spacecraft, and activate the equipment the Remnant gave you. Slowly, a large robotic claw, padded with a thick gray insulation, emerges out of your cargo hold. It takes a few tries, but you're finally able to put the satellite into your payload bay and to the fittings prepared to keep it in place.`

Reviewer: ```suggestion
			`You approach the spacecraft, and activate the equipment the Remnant gave you. Slowly, a large robotic claw, padded with a thick gray insulation, emerges out of your cargo hold. It takes a few tries, but you're finally able to put the satellite into your payload bay and get the fittings to to keep it in place prepared.`
```

				launch

mission "Ka'het: Voice From The Past 4"
	landing
	name "Return Gavriil"
	description "Bring Gavriil back to <destination>."
	source "Kumkapi"

> 	destination "Ssil Vida"

Reviewer: ```suggestion
	destination "Ssil Vida"
	deadline
```

			`	Gavriil seems enthusiastic. "Fantastic. We have already prepared the machinery delivered from Aventine, so we should be able to start the transfer right now," he signs, gesturing toward four contraptions each about as large as a closet. "I hope your journey went well."`
			choice
				`	"About as well as a mission in the middle of the Graveyard can go."`
				`	"It could have gone better."`
			`	"I will make sure to reward you appropriately once this is said and done," he replies. Behind him, two technicians climb atop a scaffolding and take a closer look to the satellite you brought with you. "One of the challenges with this kind of artifact is not only its alien origins, but also the era in which they were designed.`
			`	"The Builder empire lasted for tens of millennia; their technological level varied wildly during this time, often becoming incompatible with the previous one. We have done our best trying to date this spacecraft, so this is the moment of truth."`

> 			`	A technician carefully hands a wide cable to the two atop the platform, who move it toward the captured satellite and try to insert it to an opening on its back.`

Reviewer: ```suggestion
			`	A technician carefully hands a wide cable to the two atop the platform, who move toward the captured satellite and try to insert it into an opening on its back.`
```

> 			`	It takes a few attempts, but at last they seem to succeed. While you can't quite see what one technician is signing from the control console, it must be great news judging from the cheer that erupts right after.	"The transfer has started!" Gavriil gestures in celebration. He rushes toward the terminals at the back of the hangar, almost in disbelief.`

Reviewer: ```suggestion
			`	It takes a few attempts, but at last they seem to succeed. While you can't quite see what one technician is signing from the control console, it must be great news judging from the cheer that erupts right after. "The transfer has started!" Gavriil gestures in celebration. He rushes toward the terminals at the back of the hangar, almost in disbelief.`
```

			`	As hours pass, the excitement only partly wears off given the sheer amount of data taken from the spacecraft. Finally, Gavriil comes back to you with anticipation. "The transfer is mostly complete by now, and the analysis will soon begin. If you are ready, this would be a good time to return to Ssil Vida - it will take days before any result is ready."`
				accept

mission "Ka'het: Voice From The Past 5"
	landing

> 	name "Keep going"

Reviewer: This name could probably use some expansion.

Author: Changed to Return to <planet>

	source "Ssil Vida"
	destination "Kumkapi"
	to offer
		has "Ka'het: Voice From The Past 4: done"
	on offer
		conversation
			`During the trip Gavriil has received a constant stream of communications, either from Kumkapi or Ssil Vida. He walks alongside you out of the airlock and down the ramp, silent but clearly thoughtful.`
			`	"Thank you for this, Captain - not just for ferrying me back, but also for making this little adventure possible in first place. There is still so much to discover about the Builders and their civilization, and I am happy to have contributed to this puzzle at least a tiny bit.`
			`	"One of the key events in order to understand their history is the first contacts they had with alien species. As it often happens, the first one was with the peaceful Quarg, but there is at least another encounter we know of that did not go nearly as well.`
			`	"We do not know whether this was an actual invasion attempt or just an explorer that did not react well under pressure, but the two species fought, and the Builders only won after multiple days of fighting. The militarization of the Builder society would start at the same time."`

> 			`	"What species were they attacked by?" you ask. "We cannot say this for certain. From what I understand, no alien remains were found during the past Graveyard expeditions - other than Builder ones, naturally. However, an interstellar spanning species does not just disappear without leaving any trace, and there is only a small number of candidates that fits the bill."`

Reviewer: ```suggestion
			`	"What species were they attacked by?" you ask. "We cannot say this for certain. From what I understand, no alien remains were found during the past Graveyard expeditions - other than Builder ones, naturally. However, an interstellar spanning species does not just disappear without leaving any trace, and there are only a small number of candidates who fit the bill."`
```

			choice
				`	"The Korath?"`
					goto korath
				`	"The Hai?"`
				`	"The Sheragi?"`
			`	"A good guess. Both the Hai and the Sheragi would make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard. I heard there are great debates about the two species going on to this day, especially after the discoveries in Ssil Vida, but I am not the person to ask about them.`
				goto continue
			label korath

> 			`	"Not this time," he signs amused. "More time has passed between the end of the Builders and the Korath's discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi could make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Reviewer: ```suggestion
			`	"Not this time," he signs, amused. "More time has passed between the end of the Builders and the Korath's discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi could make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
```

			label continue

> 			`	"Whatever happened, the scars left by this short encounter were not merely psychological, as this satellite shows. Some kind of virus started spreading soon after the first contact, possibly of alien origin. We do not know the impact it had on their civilization, but it must have been quite severe: Alix appears to have been used as a quarantine facility at the time, and satellites like this had the sole function of warning incoming ships of the danger."`

Reviewer: ```suggestion
			`	"Whatever happened, the scars left by this short encounter were not merely psychological, as this satellite shows. Some kind of virus started spreading soon after the first contact, possibly of alien origin. We do not know the impact it had on their civilization, but it must have been quite severe: Alix appears to have been used as a quarantine facility at the time, and satellites like this had the sole function of warning incoming ships of the danger.`
```

> 			`	"If you were to find anything else, do not hesitate to tell us. The analysis is still ongoing, but I have been able to spare 300,000 credits to deposit on your account to show our gratitude."`

Reviewer: ```suggestion
			`	"If you find anything else, do not hesitate to tell us. The analysis is still ongoing, but I have been able to spare 300,000 credits to deposit on your account to show our gratitude."`
```
Consider using `<payment>` here.

Author: I don't think it would work with the payment only being applied in one of the branches

			`	"Farewell, Captain, and may the Embers guide you back soon."`
				accept
			label capitalist
			action
				payment 300000
				event "ka'het: quarantine site" 80
			`	"Farewell, Captain, and may the Embers guide you back soon," he gestures with a short bow.`
				decline

mission "Ka'het: Voice From The Past 6"
	landing
	name "Explore <stopovers>"
	description "Visit <destination> to search for the remains of the quarantine facility."
	source "Kumkapi"
	stopover Alix
	to offer
		has "Ka'het: Voice From The Past 5: done"
	on offer
		conversation
			`<planet> looks far more active than it was just a few days ago. There is a Remnant ship in almost every landing pad close to the outpost, and there are even a couple Albatrosses in the middle of the sitting fleet. Luckily, you find a pad reserved for your arrival right next to the main entrance.`
			`	Before you have even reached the bottom of your ship's stairway, a group of locals has gathered below your ship with anticipation. You attempt to salute them, but they seem to be more interested in your ship: a few of them pry open a panel near your cargo bay, trying to connect with the payload controls.`

> 			`	You try to ask for explanations, but they redirect you in the direction of the pad manager. Finally, after spending what felt like an hour trying to make your way through the crowd, you find her handling the loading operations at the back of a Merganser.`

Reviewer: ```suggestion
			`	You try to ask for explanations, but they direct you toward the pad manager. Finally, after spending what feels like an hour trying to make your way through the crowd, you find her handling the loading operations at the back of a Merganser.`
```

			choice
				`	(Ask her about your mission.)`
				`	"What are they doing with my ship?"`

> 			`	"It seems there must have been some miscommunications along the way," she signs in reply to your queries. "Sorry, but we have not anything close to this traffic for at least the past thirty years. We cannot afford to waste even a single minute right now.`

Reviewer: ```suggestion
			`	"It seems there must have been some miscommunications along the way," she signs in reply to your queries. "Sorry, but we have not had anything close to this amount of traffic for at least the past thirty years. We cannot afford to waste even a single minute right now.`
```

			`	"Ever since someone bought us that satellite a few days ago," she continues, seemingly oblivious about who you are, "it has been a constant stream of starships wanting to take a look at it and do all kinds of analysis. We are not staffed for this kind of traffic, not even close."`
			`	You try to inquire further about your mission, but she does not seem to know anything more than you. She redirects you again to the pad, pointing toward the bill of lading attached to the cargo they are mounting on your ship.`
			`	At last, you find in there the information you needed: your payload is a large penetrating scanner, and your task is to use it just above the surface of <stopovers> in a location near the south pole of that moon to try to locate the quarantine facility.`
				accept
	on stopover
		conversation

> 			`Unlike many of the moons you have visited in the Ember Waste, nothing about Alix stands out as particularly unique. It is a small, rocky moon orbiting close above the Pathera gas giant, nearly atmosphereless and showing an appropriately low gravity. Its surface is punctured by millions of small meteoroid impacts, certainly attracted by Pathera's deep gravity well. Even New Boston had a couple giant planets with moons in every way similar to this one.`

Reviewer: New Boston had giant planets?

Author: Ruticulus has two gas giants, I used New Boston since I imagined the player would be more likely to remember that name

> 			`	The Builders seem to have found some interesting uses for it, however. Among the destroyed settlements all around Alix, vast mining facilities not only survived the attacks, but still see a constant stream of Ka'mar vehicles coming to gather their mineral cargo and deliver it to the imposing Ka'het station orbiting the same star`

Reviewer: ```suggestion
			`	The Builders seem to have found some interesting uses for it, however. Among the destroyed settlements all around Alix, vast mining facilities not only survived the attacks, but still see a constant stream of Ka'mar vehicles coming to gather their mineral cargo and deliver it to the imposing Ka'het station orbiting the same star.`
```

			`	You pitch down and lower your altitude to look for debris, but you're not able to find anything useful. There were many ruins on the way here that could have certainly been a quarantine site at some point, but there's just nothing here, as if it was never even inhabited in first place.`
			`	Perplexed, you turn your ship around and activate the high-power scanners the Remnant gave you. The task is very tedious: you have to pass over every bit of terrain, going just at the right speed to allow the instruments to function correctly and careful not to skew their data with the wrong movement.`
			`	It takes an hour, but a familiar beep from the cockpit panel finally signals you can come back to <destination> to deliver the data.`
	on complete
		event "ka'het: quarantine site" 25
		conversation

> 			`Unlike your last visit, the testing range on <planet> has returned eerily empty once again. Gone are the massive fleets occupying every parking spot, and the few that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`

Reviewer: ```suggestion
			`Unlike your last visit, the testing range on <planet> has returned to an eerily empty condition once again. Gone are the massive fleets occupying every parking spot, and the few ships that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`
```

			set "builder signal"
		on kill
			dialog
				`A glance at the radar screen reveals that a nearby Builder satellite just blew up, presumably due to the damage incurred. A wasted opportunity.`
			clear "builder signal"

mission "Ka'het: Voice From The Past 1"
	name "Builder signal"
	description "Look for Gavriil aboard <destination> to ask him about the Builder satellite."
	source
		government "Remnant"
		attributes "remnant primary"
	destination "Ssil Vida"
	to offer
		has "builder signal"
	on offer
		conversation
			`Do you want to ask the Remnant about the communication received from the Builder satellite?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
			`	You spend at least half an hour looking for someone in the spaceport, but you can't seem to find anyone familiar. Just as you're starting to lose hope, you notice a flurry of movement in the direction of one of the scientific laboratories.`

> 			`	Moving closer, you notice the crowd moving away to leave space for a convoy of heavy machinery to pass through. Though you can't seem to identify their function, you do recognize the person leading them: it's Plume, the scientist leading the Void Sprite research program.`

Author: Added

			choice
				`	(Try to catch his attention.)`
			`	You wave your arm to try to catch his attention as you slowly squeeze through the crowd. "Captain <first>! I did not expect to see you here. You were part of the fleet as well?" he signs, gesturing toward the cargo being transported with excitement.`

> 			`	Seeing your confused expression, he understands his mistake. "Sorry, I have been having spending some long days in the laboratory recently," he continues with a slight embarrassment. "It is good that we have such an abundance of data, mind you, but the preparations for this expedition have been exhausting."`

Author: Would removing the "Sorry, " do enough?

> 			`	Seeing your confused expression, he understands his mistake. "Sorry, I have been having spending some long days in the laboratory recently," he continues with a slight embarrassment. "It is good that we have such an abundance of data, mind you, but the preparations for this expedition have been exhausting."`

Reviewer: Probably. You'll also want to tweak the second sentence from "with a slight" to "with slight"

			choice
				`	"What are you transporting?"`
				`	"I have found something interesting in Fereti."`
					goto satellite

> 			`	"Cryogenic vats, full of Void Sprite material. All the work done in Esquiline is finally bearing its fruits, and this is the more noticeable result. It did not take long after we left them to roam in Pantica that we started to realize their close passes to the station left visible traces of diaphoresis all over it. All we had to do was put detachable panels over its surface, so that we could start gathering and analyzing those samples.`

Reviewer: Will the average player know what "diaphoresis" means? Might be better to replace it with a more commonly-known word.

Author: Probably not, but I felt like that better fit with the technical langued used by Plume occasionally. I would be open to alternatives

Author: I think secretion fits better yeah, molting risks implying the Wanderer process for people not too familiar with English

			`	"Of course, most of this machinery is dedicated to keeping the material at the right conditions of pressure, temperature and composition. Each vat contains just a few kilograms of samples inside of it, at most. Still, it is hard to overstate the impact such a continuous and harmless source of Void Sprite samples could have on genetic research over time.`
			`	"What about you?"`
			label satellite
			`	You explain Plume about your encounter with the Builder satellite and the signal it sent you. By the end of it, however, he still seems quite confused. "I was hoping there would be Void Sprites involved," he replies jokingly, "but it was an interesting story nonetheless.`
			`	"I am not quite sure how to help you this time, though - none of this is really within my expertise."`
			choice
				`	"Who should I ask then?"`
				`	"That's a shame."`

> 			`	Plume stops to oversee a group of technicians unloading the content of the first vehicle, gesturing you to wait for him. He comes back to you a few minutes later. "I have had an idea. Follow me."`

Reviewer: ```suggestion
			`	Plume stops to oversee a group of technicians unloading the content of the first vehicle, gesturing you to wait for him. He comes back to you a few minutes later. "I have had an idea. Follow me.`
```

				accept
	on accept
		fail "Ka'het: Voice From The Past 0"
	on complete
		conversation
			`The more time passes, the more Ssil Vida starts to look like a Remnant outpost. The local spaceport, except for the sheer size of each room, has followed a transformation similar to Esquiline, with walls covered by mazes of piping and dozens of active work stations expanding the already operational outfitter. There is no need to wander around, however: Gavriil is waiting for you right beside the landing pad.`
			`	"Happy to see you again," he gestures warmly. "I hope the Embers blessed your journey."`
			choice
				`	"They did."`
				`	(Tell him about the encounter.)`
			`	You take him aside and start explaining again your encounter with the Builder satellite, but he stops you. "Plume told me everything a few days ago. It is always delightful to see something one has theorized about become reality," he continues proudly. "What you have observed is something usually called 'zombie satellite'. Not too technical of an expression maybe, but certainly effective.`

> 			`	"There are many ways a spacecraft can die, some of which less violent than others. And there are times, years after the supposed end of operations, in which the spacecraft spontaneously comes back alive and communicates again with the ground. It is not a predictable event, nor it does happen only to specific designs; it simply happens, no matter if the satellite was built on Aventine, by the Hai, or through Syndicate Shipyards.`

Reviewer: The Remnant left during the Alpha Wars, but according to the HistoricalTimeline wiki page the Syndicate got started two hundred years later. This should probably be updated to account for that - maybe use "in the Deep"?

			choice
				`	"How did it come back to life?"`
				`	"Can you take a look at the signal it sent?"`
					goto signal
			`	"I have a few ideas, but there is no way to be sure. One of the most frequent causes is the software successfully rebooting after a memory failure, practically resetting it to the factory setup, but without more information this is at best a guess on my part."`
			label signal
			`	You hand him a data chip with the communication you received from the satellite, which he quickly puts into his pocket. "I will try to take a few hours off to give it a first pass with the instruments we have here. While they are not the best suited for this sort of analysis, I remember a few tricks to try to get something usable out of it," he concludes.`
			`	"In the meanwhile, feel free to take a look at the changes we have been making here. I will give you a call from the spaceport when I have something to show."`

mission "Ka'het: Voice From The Past 2"
	name "Outpost in Kumkapi"
	description "Bring Gavriil to <destination> to prepare for the capture the Builder satellite."

> 	blocked `This mission requires at least the bunk space to carry Gavriil with you.`

Reviewer: ```suggestion
	blocked `This mission requires enough bunk space to carry Gavriil with you.`
```

			label continue
			`	"What we have discovered in the analysis is that the communication you received contains some kind of audio recording, the meaning of which remains a mystery. I have transmitted it to Aventine so that xenolinguists will be able to archive it for further studies, but the purpose of this signal, and consequently of the satellite, will remain unknown."`
			`	He pauses for a few seconds, thoughtful. "The data we have gathered shows that the satellite is not too large, nor particularly heavy. A way to circumvent these issues could be to capture the Builder spacecraft and bring it to one of our laboratories - if it is still alive it should be possible to extract its data mostly intact, if not fully.`
			`	"Have you ever been on <planet>?" he asks, without stopping to wait for an answer. "It is a small semi-abandoned colony in <system>, orbiting right above Viminal. I used to know some people there who should have just the equipment a project like this would need."`
			choice

> 				`	"Alright, let's do this"`

Reviewer: ```suggestion
				`	"Alright, let's do this."`
```

				`	"Will you come with me?"`
			`	You make a gesture toward your ship, to which he replies with an undecided expression. "Our backlog here has been growing considerably, and leaving for another adventure right now would not make it shorter." He stops for a couple seconds, then continues.`
			`	"On the other hand, we have also been completing milestones ahead of schedule lately, and there are plenty of experienced people here who could take over for some time. I can afford to leave the reigns to someone else for a few days.`
			`	"Plus," he concludes with a smile, "it will be an opportunity to see some faces I have not met in a few years, and reminisce about the past for a bit. I should have been doing this for a while now."`
				accept

mission "Ka'het: Voice From The Past 3"
	landing
	name "Builder satellite"
	description "Capture the Builder satellite in <waypoints> and bring it to <destination>."
	blocked `The cargo is ready to load, but you do not have enough free space. Come back when you have <tons> of free space.`
	source "Kumkapi"

> 	waypoint Fereti

Reviewer: ```suggestion
	waypoint "Fereti"
```

			`	Following the landing trajectory, you steer left after passing over a canyon and prepare for the final approach. You notice his eyes light up when the target outpost comes into view, just for an instant. "So as not to leave it to rot, a few projects to reuse the abandoned facilities were given the go ahead. The one we are going to visit today has become a major testing facility for advanced technologies, may those be built by us or recovered from other sources."`
			choice
				`	"I see."`
					goto skip
				`	"Was the Key Stone search successful?"`
			`	"Not as much as we were hoping. The only other place to turn a positive result was Eavine, a gas giant's moon located right in the middle of Ka'het-controlled space. It was a scientifically interesting finding, but not one that we could make use of in the near future."`
			label skip
			`	By the time you complete the post-landing checklist, Gavriil has already left the ship your ship to meet up with the people on the ground. The crowd that has been gathering around the pad give him a warm welcome, to which he replies cheerfully. He must have spent a lot of time on this moon when he was younger.`

> 			`	After some brief introductions, you follow the group as it moves down a few ramps of stairs and inside some well-lit tunnels right below the outpost. At the end of them lies a large hangar, wide in both directions but with a rather low ceiling. "We have a few rigs we used to capture test satellites in the past," one of the technicians explains. "They were Remnant spacecrafts in that case, but there is nothing fundamentally preventing their use in your mission.`

Reviewer: ```suggestion
			`	After some brief introductions, you follow the group as it moves down a few ramps of stairs and inside some well-lit tunnels right below the outpost. At the end of them lies a large hangar, wide in both directions but with a rather low ceiling. "We have a few rigs we used to capture test satellites in the past," one of the technicians explains. "They were Remnant spacecraft in those cases, but there is nothing fundamentally preventing their use in your mission.`
```

			`	"Connecting with the spacecraft after you carry it here is going to be a bit more complicated. We have the machines necessary to keep it stored, but I had to order some special devices from Aventine to download its data after all is said and done. Either way, a fleet will have brought it here before you are back, so it should not pose a problem."`
			`	You help them load the satellite capture equipment in your ship, and make sure to leave enough space for the spacecraft itself when it will be time. "I will stay here to help them prepare for the satellite," Gavriil signs, "and hopefully remain updated with Ssil Vida on the progress made in my absence. See you in a couple weeks."`
				accept
	npc assist
		government "Builder"
		system "Fereti"
		personality derelict waiting target
		ship "Builder Satellite" "Valii'sei"

> 		conversation
> 			`You approach the spacecraft, and activate the equipment the Remnant gave you. Slowly, a large robotic claw, padded with a thick gray insulation, emerges out of your cargo hold. It takes a few tries, but you're finally able to put the satellite into your payload bay and get the fittings to keep it in place prepared.`
> 				launch

Reviewer: This is set up pretty well. Good idea!
```suggestion
		conversation
			`You approach the spacecraft, and activate the equipment the Remnant gave you. Slowly, a large robotic claw, padded with a thick gray insulation, emerges out of your cargo hold. It takes a few tries, but you're finally able to put the satellite into your cargo bay and get the fittings to keep it in place prepared.`
				launch
```

			`	Gavriil seems enthusiastic. "Fantastic. We have already prepared the machinery delivered from Aventine, so we should be able to start the transfer right now," he signs, gesturing toward four contraptions each about as large as a closet. "I hope your journey went well."`
			choice
				`	"About as well as a mission in the middle of the Graveyard can go."`
				`	"It could have gone better."`
			`	"I will make sure to reward you appropriately once this is said and done," he replies. Behind him, two technicians climb atop a scaffolding and take a closer look to the satellite you brought with you. "One of the challenges with this kind of artifact is not only its alien origins, but also the era in which they were designed.`
			`	"The Builder empire lasted for tens of millennia; their technological level varied wildly during this time, often becoming incompatible with the previous one. We have done our best trying to date this spacecraft, so this is the moment of truth."`
			`	A technician carefully hands a wide cable to the two atop the platform, who move toward the captured satellite and try to insert it into an opening on its back.`

> 			`	It takes a few attempts, but at last they seem to succeed. While you can't quite see what one technician is signing from the control console, it must be great news judging from the cheer that erupts right after. "The transfer has started!" Gavriil gestures in celebration. He rushes toward the terminals at the back of the hangar, almost in disbelief.`

Reviewer: ```suggestion
			`	It takes a few attempts, but at last they succeed. While you can't quite see what one technician is signing from the control console, it must be great news judging from the cheer that erupts right after. "The transfer has started!" Gavriil gestures in celebration. He rushes toward the terminals at the back of the hangar, almost in disbelief.`
```

			`	"One of the key events in order to understand their history is the first contacts they had with alien species. As it often happens, the first one was with the peaceful Quarg, but there is at least another encounter we know of that did not go nearly as well.`
			`	"We do not know whether this was an actual invasion attempt or just an explorer that did not react well under pressure, but the two species fought, and the Builders only won after multiple days of fighting. The militarization of the Builder society would start at the same time."`
			`	"What species were they attacked by?" you ask. "We cannot say this for certain. From what I understand, no alien remains were found during the past Graveyard expeditions - other than Builder ones, naturally. However, an interstellar spanning species does not just disappear without leaving any trace, and there are only a small number of candidates who fit the bill."`
			choice
				`	"The Korath?"`
					goto korath
				`	"The Hai?"`
				`	"The Pug?"`
					to display
						has "main plot completed"
					goto pug
				`	"The Sheragi?"`
				`	"The Predecessors?"`
					to display
						has "Successors: history"
					goto predecessors

> 			`	"A good guess. Both the Hai and the Sheragi would make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard. I heard there are great debates about the two species going on to this day, especially after the discoveries in Ssil Vida, but I am not the person to ask about them.`

Reviewer: ```suggestion
			`	"A good guess. Both the Hai and the Sheragi would make sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard. I heard there are great debates about the two species going on to this day, especially after the discoveries in Ssil Vida, but I am not the person to ask about them.`
```

				goto continue
			label pug

> 			`	"Interesting choice. We do know that they used to have a presence in this area at some point, but there is little use in speculating about their purposes with so little information, especially when there are more likely candidates. Either the Hai or the Sheragi could make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Reviewer: ```suggestion
			`	"Interesting guess. We do know that they used to have a presence in this area at some point, but there is little use in speculating about their purposes with so little information, especially when there are more likely candidates. Either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
```

				goto continue
			label predecessors

> 			`	He shakes his head with some disappointment. "Far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi could make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Reviewer: ```suggestion
			`	He shakes his head with some disappointment. "Far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
```

				goto continue
			label korath

> 			`	"Not this time," he signs, amused. "More time has passed between the end of the Builders and the Korath's discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi could make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Reviewer: ```suggestion
			`	"Not this time," he signs, amused. "More time has passed between the end of the Builders and the Korath's discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
```

			`	You wait for Gavriil for nearly an hour, but he doesn't show up again. Just before leaving, you notice he sent you a communication on your datapad. "<first>," it reads, "I won't be able to meet up with you again today. I apologize for the impoliteness, but there is just too much I need to catch up with in order to keep the project on target.`
			`	"Nevertheless, I was able to get in contact with the test range, and they seemed happy to get more help. I won't be able to come with you this time, but if follow the same indications I gave you last time you should have no problems finding them.`
			`	"Farewell, Captain, and may the Embers guide you back soon."`
				accept
			label capitalist
			action
				payment 300000
				event "ka'het: quarantine site" 80
			`	"Farewell, Captain, and may the Embers guide you back soon," he gestures with a short bow.`
				decline

mission "Ka'het: Voice From The Past 6"
	landing
	name "Explore <stopovers>"
	description "Visit <destination> to search for the remains of the quarantine facility."
	blocked `The cargo is ready to load, however you do not have enough free space. Come back when you have <tons> of free space.`
	source "Kumkapi"

> 	stopover Alix

Reviewer: ```suggestion
	stopover "Alix"
```

			`	Before you have even reached the bottom of your ship's stairway, a group of locals has gathered below your ship with anticipation. You attempt to salute them, but they seem to be more interested in your ship: a few of them pry open a panel near your cargo bay, trying to connect with the payload controls.`
			`	You try to ask for explanations, but they direct you toward the pad manager. Finally, after spending what feels like an hour trying to make your way through the crowd, you find her handling the loading operations at the back of a Merganser.`
			choice
				`	(Ask her about your mission.)`
				`	"What are they doing with my ship?"`
			`	"It seems there must have been some miscommunications along the way," she signs in reply to your queries. "Sorry, but we have not had anything close to this amount of traffic for at least the past thirty years. We cannot afford to waste even a single minute right now.`

> 			`	"Ever since someone bought us that satellite a few days ago," she continues, seemingly oblivious about who you are, "it has been a constant stream of starships wanting to take a look at it and do all kinds of analysis. We are not staffed for this kind of traffic, not even close."`

Reviewer: ```suggestion
			`	"Ever since someone brought us that satellite a few days ago," she continues, seemingly oblivious about who you are, "there have been a constant stream of starships wanting to take a look at it and do all kinds of analysis. We are not staffed for this kind of traffic, not even close."`
```

			`	You try to inquire further about your mission, but she does not seem to know anything more than you. She redirects you again to the pad, pointing toward the bill of lading attached to the cargo they are mounting on your ship.`
			`	At last, you find in there the information you needed: your payload is a large penetrating scanner, and your task is to use it just above the surface of <stopovers> in a location near the south pole of that moon to try to locate the quarantine facility.`

> 				accept

Reviewer: Optional, but the dialog mentions there being lots of ships. Perhaps add some:
```suggestion
				accept
	npc
		government "Remnant"
		personality heroic launching lingering
		fleet "Large Remnant"
		fleet "Small Remnant"
```

Author: Should they also have uninterested?

			choice
				`	(Fly closer to the mining outposts to investigate.)`
				`	(Move to the target the Remnant gave you.)`
					goto quarantine
			`	You pitch down your ship and steer toward the nearest outpost: a long series of short buildings with angled roofs, reminiscent of a solar field in the black and white screen in your ship. After coming closer, however, you realize your initial guess was wrong: these are not akin to roofs, but to sails, composed of a pale white textile. A thin layer of dust covers their surface, mostly gathered toward the bottom.`
			`	You try to come closer, but suddenly the compass on your ship goes crazy. Alarmed, you swiftly reverse course and move away from the facility, and the instrumentation soon returns nominal again after a reboot. The automated outpost is clearly emitting a strong magnetic field, but the connection among this all isn't immediately obvious.`
			choice
				`	(Do a close flyby, even if the compass goes haywire.)`
				`	(Try to search around the facility, keeping a safe distance.)`
					goto coward

> 				`	(Give up and move to the original targets.)`

Reviewer: ```suggestion
				`	(Give up and move to the original target.)`
```

					goto quarantine
			`	A brief exploration doesn't turn up anything you hadn't already seen earlier, and all you can do is watch as the mesmerizing flow of Faes'mar in and out of the facility continues undisturbed below you.`
			label coward
			`	Just as you're starting to feel this mystery will remain unsolved, you have a stroke of luck: a small meteorite hits the surface a few kilometers away from you, throwing a cloud of debris in every direction. You check the composition of the nearly imperceptible atmosphere near the surface, and find it composed almost entirely of particles of precious metals.`
			`	The overall picture has become clearer: the frequent asteroid impacts on a moon like this raise enough meteorite dust to effectively shroud the planet in an extremely sparse atmosphere, which the Builder machines attract and gather so that they can be transported off-planet. It's a slow and cumbersome process, but it's also one that was able to work for tens of thousands of years without maintenance.`
			label quarantine

> 			`	You record a brief log of your thoughts, and move on to your actual destination. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, the map eventually signals you have reached your destination: a plain, unremarkable plateau extending for kilometers in every directions.`

Reviewer: ```suggestion
			`	You record a brief log of your thoughts, and move on to your actual destination. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, the map eventually signals you have reached your destination: a plain, unremarkable plateau extending for kilometers in every direction.`
```

> 			`	You pitch down and lower your altitude to look for debris, but you're not able to find anything useful. There were many ruins on the way here that could have certainly been a quarantine site at some point, but there's just nothing here, as if it was never even inhabited in first place.`

Reviewer: ```suggestion
			`	You pitch down and lower your altitude to look for debris, but you're not able to find anything useful. There were many ruins on the way here that could have certainly been a quarantine site at some point, but there's nothing here, as if it was never even inhabited in first place.`
```
Alternatively, if you want to keep emphasis:
```suggestion
			`	You pitch down and lower your altitude to look for debris, but you're not able to find anything useful. There were many ruins on the way here that could have certainly been a quarantine site at some point, but there's absolutely nothing here, as if it was never even inhabited in first place.`
```

Author: Went with the second one

> 			`	Perplexed, you turn your ship around and activate the high-power scanners the Remnant gave you. The task is very tedious: you have to pass over every bit of terrain, going just at the right speed to allow the instruments to function correctly and careful not to skew their data with the wrong movement.`

Reviewer: ```suggestion
			`	Perplexed, you turn your ship around and activate the high-power scanners the Remnant gave you. The task is very tedious: you have to pass over every bit of terrain going just at the right speed, careful not to skew the data with the wrong movement, to allow the instruments to function correctly.`
```

			`	"They relented!" she signs excitedly after noticing you. "It took a few days, but they finally accepted that we do not have the facilities nor the personnel to handle such traffic, and moved the satellite to a more appropriate laboratory. Somewhere on Aventine, I believe."`
			choice
				`	"Is there someone I can give this data to?"`
				`	"The location you gave me did not seem to have been inhabited."`
			`	You give her a brief summary of your mission, and of your findings. "I am afraid the people you were supposed to deliver this data to have left together with the satellite," she replies perplexed. "It has been some busy days for all of us recently, and some steps were skipped to accelerate the process.`
			`	"If you leave the datapad here, I promise I will make sure it gets into the right hands. I am sure they will contact you themselves after things have calmed down a bit." You hand her the tablet with the info you recorded, hopeful it will reach its destination, and prepare to leave once again.`

> event "ka'het: quarantine site"

Reviewer: I assume you're not going to add to this in this PR?

Author: The storyline is meant as a two-parter, with the latter part using different characters and requiring Patir 2 (the 'part cycler' throwaway mention in it). This will be the timer for it to start when I get to it

			`	He pauses for a few seconds, thoughtful. "The data we have gathered shows that the satellite is not too large, nor particularly heavy. A way to circumvent these issues could be to capture the Builder spacecraft and bring it to one of our laboratories - if it is still alive it should be possible to extract its data mostly intact, if not fully.`
			`	"Have you ever been on <planet>?" he asks, without stopping to wait for an answer. "It is a small semi-abandoned colony in <system>, orbiting right above Viminal. I used to know some people there who should have just the equipment a project like this would need."`
			choice
				`	"Alright, let's do this."`
				`	"Will you come with me?"`
			`	You make a gesture toward your ship, to which he replies with an undecided expression. "Our backlog here has been growing considerably, and leaving for another adventure right now would not make it shorter." He stops for a couple seconds, then continues.`

> 			`	"On the other hand, we have also been completing milestones ahead of schedule lately, and there are plenty of experienced people here who could take over for some time. I can afford to leave the reigns to someone else for a few days.`

Reviewer: When riding a camel, you use reins, and the Remnant use camels in their remote spaceports.

			`	"One of the key events in order to understand their history is the first contacts they had with alien species. As it often happens, the first one was with the peaceful Quarg, but there is at least another encounter we know of that did not go nearly as well.`
			`	"We do not know whether this was an actual invasion attempt or just an explorer that did not react well under pressure, but the two species fought, and the Builders only won after multiple days of fighting. The militarization of the Builder society would start at the same time."`
			`	"What species were they attacked by?" you ask. "We cannot say this for certain. From what I understand, no alien remains were found during the past Graveyard expeditions - other than Builder ones, naturally. However, an interstellar spanning species does not just disappear without leaving any trace, and there are only a small number of candidates who fit the bill."`
			choice
				`	"The Korath?"`
					goto korath
				`	"The Hai?"`
				`	"The Pug?"`
					to display
						has "main plot completed"
					goto pug

> 				`	"The Sheragi?"`

Author: I assumed the player would be familiar with them given it's post-cognizance, but you're right those missions also have similar branches. I'll add it

				`	"The Predecessors?"`
					to display
						has "Successors: history"
					goto predecessors
			`	"A good guess. Both the Hai and the Sheragi would make sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard. I heard there are great debates about the two species going on to this day, especially after the discoveries in Ssil Vida, but I am not the person to ask about them.`
				goto continue
			label pug

> 			`	"Interesting guess. We do know that they used to have a presence in this area at some point, but there is little use in speculating about their purposes with so little information, especially when there are more likely candidates. Either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Author: Either Kaeyin or the Thrasher being based on the Seeker design should be big enough clues for this dialogue

				goto continue
			label predecessors

> 			`	He shakes his head with some disappointment. "Far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Reviewer: The Successors are supposed to have visited Earth during the 1500s; perhaps the Remnant were in contact with them at the time and learned this then?

> 			`	He shakes his head with some disappointment. "Far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Author: Changed to ""I may not know much on them, but I can tell you that far more time has passed between..." to make their knowledge of the Predecessors more ambiguous

> 			`	He shakes his head with some disappointment. "Far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Reviewer 2: Apologies, I was a bit vague. I meant that for the Remnant to know of the Successors and the name "Successors" from you mentioning the Predecessors, that would mean that the Remnant definitively knew about the current day Successors existing, have communicated with the Successors or intercepted/translated their language to learn the name "Successors," and were probably taught Successor history to get their connection directly back to the Predecessors.

I don't think the Remnant would know the Successors from their 1500s visit any more than other humans, since they were just a regular part of humanity until the Alpha Wars in 2450-ish.

How's something like this, so there's a bit of plausible incredulity for the Remnant learning the name Predecessor, and then the comment is about the timing from Builders to the first Predecessor-related artifacts they found, rather than relating to the modern-day Successors?

`	"Those that came before?" He shakes his head with some disappointment. "I may not know much about them, but I can tell you that far more time has passed between the end of the Builders and the first Predecessor artifacts than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

> 			`	He shakes his head with some disappointment. "Far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Author: Sounds good to me, changed

			`	You try to ask for explanations, but they direct you toward the pad manager. Finally, after spending what feels like an hour trying to make your way through the crowd, you find her handling the loading operations at the back of a Merganser.`
			choice
				`	(Ask her about your mission.)`
				`	"What are they doing with my ship?"`
			`	"It seems there must have been some miscommunications along the way," she signs in reply to your queries. "Sorry, but we have not had anything close to this amount of traffic for at least the past thirty years. We cannot afford to waste even a single minute right now.`
			`	"Ever since someone brought us that satellite a few days ago," she continues, seemingly oblivious about who you are, "there have been a constant stream of starships wanting to take a look at it and do all kinds of analysis. We are not staffed for this kind of traffic, not even close."`

> 			`	You try to inquire further about your mission, but she does not seem to know anything more than you. She redirects you again to the pad, pointing toward the bill of lading attached to the cargo they are mounting on your ship.`

Reviewer 2: ```suggestion
			`	You try to inquire further about your mission, but she does not seem to know any more than you do. She redirects you again to the pad, pointing toward the bill of landing attached to the cargo they are mounting on your ship.`
```

			`	At last, you find in there the information you needed: your payload is a large penetrating scanner, and your task is to use it just above the surface of <stopovers> in a location near the south pole of that moon to try to locate the quarantine facility.`
				accept
	npc
		government "Remnant"
		personality heroic launching lingering
		fleet "Large Remnant"
		fleet "Small Remnant"
	on stopover
		conversation

> 			`Unlike many of the moons you have visited in the Ember Waste, nothing about Alix stands out as particularly unique. It is a small, rocky moon orbiting close above the Pathera gas giant, nearly atmosphereless and showing an appropriately low gravity. Its surface is punctured by millions of small meteoroid impacts, certainly attracted by Pathera's deep gravity well. Even New Boston had a couple of gas giants with moons in every way similar to this one.`

Reviewer 2: Probably needs something to check the player's start, for Hai start & future starts.

Author: What should I check for that?

Reviewer: `has "human space start"` or `not "hai space start"`

Author: Added

	"reputation: Ka'sei" = 1

event "patir mystery timer"

ship "Builder Satellite"
	sprite "ship/builder satellite"
	attributes
		category "Drone"
		"hull" 150
		"mass" 10
		"drag" 2
		"heat dissipation" 1

ship "Builder Satellite" "Builder Satellite 2"
	explode "tiny explosion" 5
	explode "small explosion" 2

mission "Ka'het: Voice From The Past 0"
	landing
	invisible

> 	minor

Author: Removed, there's not really much reason for it given the chain does not start until the player chooses to board it

			`	"This pad contains the extracts of all our published engineering papers - the publicly available ones, at least. If what you have encountered has been studied previously, it should be here somewhere." Plume returns to his job after showing you the basics, signing to give him a call if you need more help.`
			`	Left by yourself, you grab a nearby seat and start making some searches inside the terminal. The first half hour does not return any useful result: there are millions of papers to choose from, and the initial queries you come up with find either thousands of possible matches, or none at all.`
			`	Eventually, however, you start getting the hang of it. You find one paper that describes the phenomena you observed, and another, and more. By the end of the hour, you have ended up with and saved at least a dozen relevant papers.`
			`	Though you can't understand any of them in depth, a detail catches your eye: three of those papers were authored by Gavriil, the engineer you last left in <system>. If there's anyone who can get you to the end of this, it has to be him.`

> 			`	You wave goodbye to Plume, still busy transferring the delicate cargo to the safety of the laboratory, and climb aboard your ship. Next stop: <planet>.`

Author: Added

			choice
				`	"Alright, let us do this."`
				`	"Will you come with me?"`
			`	You make a gesture toward your ship, to which he replies with an undecided expression. "Our backlog here has been growing considerably, and leaving for another adventure right now would not make it shorter." He stops for a couple seconds, then continues.`
			`	"On the other hand, we have also been completing milestones ahead of schedule lately, and there are plenty of experienced people here who could take over for some time. I can afford to leave the reins to someone else for a few days.`
			`	"Plus," he concludes with a smile, "it will be an opportunity to see some faces I have not seen in years, and to reminisce about the past for a bit. I should have done this a while ago."`
				accept

mission "Ka'het: Voice From The Past 3"
	landing
	name "Builder satellite"
	description "Capture the Builder satellite in <waypoints> and bring it to <destination>."
	blocked `The cargo is ready to load, but you do not have enough free space. Come back when you have <tons> of free space.`
	source "Kumkapi"
	waypoint "Fereti"
	cargo "Satellite Capture Equipment" 45

> 	to offer

Author: The player could have a JD and complete the trip in just 2 weeks, so I wouldn't change the text

event "patir mystery timer"

ship "Builder Satellite"
	sprite "ship/builder satellite"
	attributes
		category "Drone"
		"hull" 150
		"mass" 10
		"drag" 2
		"heat dissipation" 1

ship "Builder Satellite" "Builder Satellite 2"
	explode "tiny explosion" 5
	explode "small explosion" 2

mission "Ka'het: Voice From The Past 0"
	landing
	invisible
	minor
	to offer
		has "event: remnant: cognizance calm"
	npc assist save
		government "Builder"
		system "Fereti"
		personality derelict target waiting
		ship "Builder Satellite 2" "Valii'sei"
		on assist
			dialog
				`You make a close pass above the lifeless satellite, abandoned in a graveyard orbit tens of thousands of years ago. All of a sudden, your commlink starts emitting a weak, garbled whistle-like sound, as if receiving a communication from the Builder spacecraft. While you can't understand it, it might be worth it to pay another visit to the Remnant.`
			set "builder signal"
		on kill
			dialog
				`A glance at the radar screen reveals that the nearby Builder satellite exploded, presumably due to the damage incurred. A wasted opportunity.`
			clear "builder signal"

mission "Ka'het: Voice From The Past 1"

> 	name "Builder signal"

Author: The mission has you look in the spaceport, and it will already offer on any remnant planet you look into for simplicity

	description "Look for Gavriil aboard <destination> to ask him about the Builder satellite."
	source
		government "Remnant"

> 		attributes "remnant primary"

Author: This is how the filter is normally done from what I can see, e.g. Expanded Horizons Quarg

	destination "Ssil Vida"
	to offer
		has "builder signal"
	on offer
		conversation
			`Do you want to ask the Remnant about the communication received from the Builder satellite?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer
			`	You spend at least half an hour looking for someone in the spaceport, but you can't seem to find anyone familiar. Just as you're starting to lose hope, you notice a flurry of movement in the direction of one of the scientific laboratories.`
			`	Moving closer, you notice the crowd moving away to leave space for a convoy of heavy machinery to pass through. Though you can't seem to identify their function, you do recognize the person leading them: it's Plume, the scientist leading the Void Sprite Research program.`

> 			`	You wave your arm to try to catch his attention as you slowly squeeze through the crowd. "Captain <first>! I did not expect to see you here. You were part of the fleet as well?" he signs, excitedly gesturing toward the cargo being transported.`

Author: Changed phrase to `You wave your arm to try to catch his attention as you slowly squeeze through the crowd until he notices you.`

			`	"Of course, most of this machinery is dedicated to keeping the material at the right conditions of pressure, temperature and composition. Each vat contains just a few kilograms of samples at most. Still, it is hard to overstate the impact such a continuous and harmless source of Void Sprite samples could have on genetic research over time.`
			`	"What about you?"`
			label satellite
			`	You explain to Plume about your encounter with the Builder satellite and the signal it sent you. By the end of it, however, he still seems quite confused. "I was hoping there would be Void Sprites involved," he jokes, "but it was an interesting story nonetheless.`
			`	"I am not quite sure how to help you this time, though - none of this is really within my expertise."`
			choice
				`	"Who should I ask then?"`
				`	"That is a shame."`

> 			`	Plume stops to oversee a group of technicians unloading the contents of the first vehicle, gesturing for you to wait for him. He comes back to you a few minutes later. "I have had an idea. Follow me.`

Author: Done

			`	Left by yourself, you grab a nearby seat and start making some searches inside the terminal. The first half hour does not return any useful result: there are millions of papers to choose from, and the initial queries you come up with find either thousands of possible matches, or none at all.`
			`	Eventually, however, you start getting the hang of it. You find one paper that describes the phenomena you observed, and another, and more. By the end of it, you have ended up with and saved at least a dozen relevant papers.`
			`	Though you can't understand any of them in depth, a detail catches your eye: three of those papers were authored by Gavriil, the engineer you last left in <system>. If there's anyone who can get you to the end of this, it has to be him.`
			`	You wave goodbye to Plume, still busy transferring the delicate cargo to the safety of the laboratory, and climb aboard your ship. Next stop: <planet>.`
				accept
	on accept

> 		fail "Ka'het: Voice From The Past 0"

Author: Done

			label signal
			`	You hand him a data chip with the communication you received from the satellite, which he quickly puts into his pocket. "I will try to take a few hours off to give it a first pass with the instruments we have here. While they are not the best suited for this sort of analysis, I remember a few tricks that might get something usable out of it," he concludes.`
			`	"In the meantime, feel free to take a look at the changes we have been making here. I will give you a call from the spaceport when I have something to show."`

mission "Ka'het: Voice From The Past 2"
	name "Outpost in Kumkapi"
	description "Bring Gavriil to <destination> to prepare for the capture of the Builder satellite."
	blocked `This mission requires enough one empty bunk space to carry Gavriil with you.`
	source "Ssil Vida"
	destination "Kumkapi"
	passengers 1
	deadline
	to offer
		has "Ka'het: Voice From The Past 1: done"
	on offer
		conversation

> 			`Ssil Vida's outfitter was built atop the original storage platforms of the station, far too large for the Remnant's logistics system. While it does not seem to be particularly well equipped, it's still quite impressive that they were able to set it up on completely alien infrastructure so quickly.`

Author: Reworded to `While it does not seem to be particularly well equipped, what they were able to set up on completely alien infrastructure since the first time you came here is still impressive.`

			label skip
			`	By the time you complete the post-landing checklist, Gavriil has already left the ship to meet up with the people on the ground. The crowd that has been gathering around the pad give him a warm welcome, to which he replies cheerfully. He must have spent a lot of time on this moon when he was younger.`
			`	After some brief introductions, you follow the group as it moves down a few ramps of stairs and inside some well-lit tunnels right below the outpost. At the end of them lies a large hangar, wide in both directions but with a rather low ceiling. "We have a few rigs we used to capture test satellites in the past," one of the technicians explains. "They were Remnant spacecraft in those cases, but there is nothing fundamentally preventing their use in your mission.`
			`	"Connecting with the spacecraft after you carry it here is going to be a bit more complicated. We have the machines necessary to keep it stored, but I had to order some special devices from Aventine to download its data after all is said and done. Either way, a fleet will have brought it here before you are back, so it should not pose a problem."`

> 			`	You help them load the satellite capture equipment in your ship, and make sure to leave enough space for the spacecraft itself when it will be time. "I will stay here to help them prepare for the satellite," Gavriil signs, "and hopefully remain updated with Ssil Vida on the progress made in my absence. See you in a couple weeks."`

Reviewer: Using "when it will be time" doesn't flow well here, and it isn't really necessary - the meaning has already been conveyed.
```suggestion
			`	You help them load the satellite capture equipment in your ship, and make sure to leave enough space for the spacecraft itself. "I will stay here to help them prepare for the satellite," Gavriil signs, "and hopefully remain updated with Ssil Vida on the progress made in my absence. See you in a couple weeks."`
```

				accept
	npc assist
		government "Builder"
		system "Fereti"

> 		personality derelict waiting target

Reviewer: ```suggestion
		personality derelict target waiting
```

	source "Kumkapi"
	destination "Ssil Vida"
	deadline
	to offer
		has "Ka'het: Voice From The Past 3: done"
	on offer
		conversation
			`A few pad workers guide you to the designated landing site, where a large harness is waiting for your special payload to arrive. Once the satellite's container has been safely berthed to its transporter, you follow the convoy as it slowly descends down a wide ramp and into the same hangar you had been shown earlier.`
			`	Gavriil seems enthusiastic. "Fantastic. We have already prepared the machinery delivered from Aventine, so we should be able to start the transfer right now," he signs, gesturing toward four contraptions - each about as large as a closet. "I hope your journey went well."`
			choice
				`	"About as well as a mission in the middle of the Graveyard can go."`
				`	"It could have gone better."`
			`	"I will make sure to reward you appropriately once this is said and done," he replies. Behind him, two technicians climb atop a scaffolding and take a closer look at the satellite you brought with you. "One of the challenges with these kinds of artifact is not only its alien origins, but also the era in which they were designed.`

> 			`	"The Builder empire lasted for tens of millennia; their technological level varied wildly during this time, often becoming incompatible with the previous one. We have done our best trying to date this spacecraft, so this is the moment of truth."`

Reviewer: ```suggestion
			`	"The Builder empire lasted for tens of millennia; their technological level varied wildly during this time, often becoming incompatible with the previous one. We have done our best to date this spacecraft, so this is the moment of truth."`
```

> 			`	A technician carefully hands a wide cable to the two atop the platform, who move toward the captured satellite and try to insert it into an opening on its back.`

Reviewer: I know I changed this to "it" originally, but it might work better like so:
```suggestion
			`	A technician carefully hands a wide cable to the two atop the platform, who move toward the captured satellite and try to insert the cord into an opening on its back.`
```

> 			`	It takes a few attempts, but at last they succeed. While you can't quite see what one technician is signing from the control console, it must be great news judging from the cheer that erupts right after. "The transfer has started!" Gavriil gestures in celebration. He rushes toward the terminals at the back of the hangar, almost in disbelief.`

Reviewer: ```suggestion
			`	It takes a few attempts, but at last they succeed. While you can't quite see what one technician is signing from the control console, it must be great news, judging from the cheer that erupts right after. "The transfer has started!" Gavriil gestures in celebration. He rushes toward the terminals at the back of the hangar, almost in disbelief.`
```

> 			`	As hours pass, the excitement only partly wears off given the sheer amount of data taken from the spacecraft. Finally, Gavriil comes back to you with anticipation. "The transfer is mostly complete by now, and the analysis will soon begin. If you are ready, this would be a good time to return to Ssil Vida - it will take days before any result is ready."`

Reviewer: ```suggestion
			`	The excitement only slightly wears off as the hours pass, the sheer amount of data taken from the spacecraft boosting it again and again. Finally, Gavriil comes back to you with anticipation. "The transfer is mostly complete by now, and the analysis will soon begin. If you are ready, this would be a good time to return to Ssil Vida - it will take days before any result is ready."`
```
I tried to reword this to flow slightly better. Let me know if you think it needs further slight adjustments.

				accept

mission "Ka'het: Voice From The Past 5"
	landing
	name "Return to <planet>"
	description "Return to <destination> to get some new equipment."
	source "Ssil Vida"
	destination "Kumkapi"
	to offer
		has "Ka'het: Voice From The Past 4: done"
	on offer
		log `Captured an active Builder satellite. It was a warning beacon orbiting the moon Alix, which appears to have been used as a quarantine site to defend against an alien plague.`
		event "ka'het: alix discoveries"
		conversation
			`During the trip Gavriil has received a constant stream of communications, either from Kumkapi or Ssil Vida. He walks alongside you out of the airlock and down the ramp, silent but clearly thoughtful.`

> 			`	"Thank you for this, Captain - not just for ferrying me back, but also for making this little adventure possible in the first place. There is still so much to discover about the Builders and their civilization, and I am happy to have contributed to this puzzle at least a tiny bit.`

Reviewer: ```suggestion
			`	"Thank you for this, Captain - not just for ferrying me back, but also for making this little adventure possible in the first place. There is still much to discover about the Builders and their civilization, and I am happy to have contributed to this puzzle at least a tiny bit.`
```

			`	"One of the key events in order to understand their history is the first contacts they had with alien species. As it often happens, the first one was with the peaceful Quarg, but there is at least another encounter we know of that did not go nearly as well.`
			`	"We do not know whether this was an actual invasion attempt or just an explorer that did not react well under pressure, but the two species fought, and the Builders only won after multiple days of fighting. The militarization of the Builder society would start at the same time."`

> 			`	"What species were they attacked by?" you ask. "We cannot say this for certain. From what I understand, no alien remains were found during the past Graveyard expeditions - other than Builder ones, naturally. However, an interstellar spanning species does not just disappear without leaving any trace, and there are only a small number of candidates who fit the bill."`

Reviewer: ```suggestion
			`	"What species were they attacked by?" you ask. "We cannot say for certain. From what I understand, no alien remains were found during the past Graveyard expeditions - other than Builder ones, naturally. However, an interstellar spanning species does not just disappear without leaving any trace, and there are only a small number of candidates who fit the bill."`
```

			choice
				`	"The Korath?"`
					goto korath
				`	"The Hai?"`
				`	"The Pug?"`
					to display
						has "main plot completed"
					goto pug
				`	"The Sheragi?"`
					to display
						has "event: rim archaeology results"
				`	"The Predecessors?"`
					to display
						has "Successors: history"

> 					goto predecessors

Reviewer: https://github.com/endless-sky/endless-sky/blob/dcf951e3b0175f0dcad409c35d0b21d55a76512d/data/successors/successor%201%20prologue.txt#L197
This condition is cleared at the end of the mission. That probably needs to be updated if you want to use it here.

Author: Changed to use "event: label successor space"

			`	You wait for Gavriil for nearly an hour, but he doesn't show up again. Just before leaving, you notice he sent you a communication on your datapad. "<first>," it reads, "I won't be able to meet up with you again today. I apologize for the impoliteness, but there is just too much I need to catch up with in order to keep the project on target.`
			`	"Nevertheless, I was able to get in contact with the test range, and they seemed happy to get more help. I won't be able to come with you this time, but if follow the same indications I gave you last time you should have no problems finding them.`
			`	"Farewell, Captain, and may the Embers guide you back soon."`
				accept
			label capitalist
			action
				payment 300000
				event "ka'het: quarantine site" 80
			`	"Farewell, Captain, and may the Embers guide you back soon," he gestures with a short bow.`
				decline

mission "Ka'het: Voice From The Past 6"
	landing
	name "Explore <stopovers>"
	description "Visit <destination> to search for the remains of the quarantine facility."

> 	blocked `The cargo is ready to load, however you do not have enough free space. Come back when you have <tons> of free space.`

Reviewer: ```suggestion
	blocked `The cargo is ready to load, but you do not have enough free space. Come back when you have <tons> of empty cargo space.`
```

	source "Kumkapi"
	stopover "Alix"

> 	cargo "High-power scanners" 7

Reviewer: ```suggestion
	cargo "High-powered scanner" 7
```

			`	Before you have even reached the bottom of your ship's stairway, a group of locals has gathered below your ship with anticipation. You attempt to salute them, but they seem to be more interested in your ship: a few of them pry open a panel near your cargo bay, trying to connect with the payload controls.`
			`	You try to ask for explanations, but they direct you toward the pad manager. Finally, after spending what feels like an hour trying to make your way through the crowd, you find her handling the loading operations at the back of a Merganser.`
			choice
				`	(Ask her about your mission.)`
				`	"What are they doing with my ship?"`
			`	"It seems there must have been some miscommunications along the way," she signs in reply to your queries. "Sorry, but we have not had anything close to this amount of traffic for at least the past thirty years. We cannot afford to waste even a single minute right now.`

> 			`	"Ever since someone brought us that satellite a few days ago," she continues, seemingly oblivious about who you are, "there have been a constant stream of starships wanting to take a look at it and do all kinds of analysis. We are not staffed for this kind of traffic, not even close."`

Reviewer: "starships" implies some sort of world ship, I feel. What would be good synonyms to use here?
I think something like "spacecraft" or "off-planeters" would work best.

Author: Changed to spacecrafts

			`	You try to inquire further about your mission, but she does not seem to know any more than you do. She redirects you again to the pad, pointing toward the bill of landing attached to the cargo they are mounting on your ship.`
			`	At last, you find in there the information you needed: your payload is a large penetrating scanner, and your task is to use it just above the surface of <stopovers> in a location near the south pole of that moon to try to locate the quarantine facility.`
				accept
	npc
		government "Remnant"
		personality heroic launching lingering
		fleet "Large Remnant"
		fleet "Small Remnant"
	on stopover
		conversation
			branch "not boston"

> 				has "hai space start"

Reviewer: #11302 
When that is merged you'll need to update here (unless VFTP is merged first)

These two paragraphs could be simplified with the use of `<starting planet>`, in that case.

Author: I can't find that substitution on the wiki, but I've changed it all to just `Even <starting planet> had a couple of gas giants with moons in every way similar to this one.` hoping it works

Reviewer 2: I think even in the current start we need the branch rather than solely a substitution, since with Hai start there's only 1 gas giant with 1 rocky moon, so that the plurals in "Even starting planet had a **couple** of gas giants with **moons** in every way similar to this one." would be incorrect in the Hai start.

			label builders
			`	The Builders seem to have found some interesting uses for it, however. Among the destroyed settlements all around Alix, vast mining facilities not only survived the attacks, but still see a constant stream of Ka'mar vehicles coming to gather their mineral cargo and deliver it to the imposing Ka'het station orbiting the same star.`
			`	Strangely enough, this moon does not show any sign of strip mining, or in fact any digging at all. After operating mining facilities for a hundred thousand years, the surface should show clearly visible signs of such an intensive operation, but there seem to be none.`
			choice
				`	(Fly closer to the mining outposts to investigate.)`
				`	(Move to the target the Remnant gave you.)`
					goto quarantine
			`	You pitch down your ship and steer toward the nearest outpost: a long series of short buildings with angled roofs, reminiscent of a solar field in the black and white screen in your ship. After coming closer, however, you realize your initial guess was wrong: these are not akin to roofs, but to sails, composed of a pale white textile. A thin layer of dust covers their surface, mostly gathered toward the bottom.`

> 			`	You try to come closer, but suddenly the compass on your ship goes crazy. Alarmed, you swiftly reverse course and move away from the facility, and the instrumentation soon returns nominal again after a reboot. The automated outpost is clearly emitting a strong magnetic field, but the connection among this all isn't immediately obvious.`

Reviewer: Maybe describe how it goes crazy in a bit more detail?

Author: Changed to `You try to come closer, but the compass on your ship suddenly starts spinning randomly!`

	on complete
		event "ka'het: quarantine site" 25
		conversation
			`Unlike your last visit, the testing range on <planet> has returned to an eerily empty condition once again. Gone are the massive fleets occupying every parking spot, and the few ships that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`
			`	With a datapad in your hand and the pad workers unloading your used mission equipment, you start looking around for someone to deliver the data to. The only people you can find nearby are the ones working on your ship, as well as the pad manager you had met earlier.`
			`	"They relented!" she signs excitedly after noticing you. "It took a few days, but they finally accepted that we do not have the facilities nor the personnel to handle such traffic, and moved the satellite to a more appropriate laboratory. Somewhere on Aventine, I believe."`
			choice
				`	"Is there someone I can give this data to?"`
				`	"The location you gave me did not seem to have been inhabited."`

> 			`	You give her a brief summary of your mission, and of your findings. "I am afraid the people you were supposed to deliver this data to have left together with the satellite," she replies perplexed. "It has been some busy days for all of us recently, and some steps were skipped to accelerate the process.`

Reviewer: ```suggestion
			`	You give her a brief summary of your mission, and of your findings. "I am afraid the people you were supposed to deliver this data to have left together with the satellite," she replies, perplexed. "It has been some busy days for all of us recently, and some steps were skipped to accelerate the process.`
```

	landing
	name "Builder satellite"
	description "Capture the Builder satellite in <waypoints> and bring it to <destination>."

> 	blocked `The cargo is ready to load, however you do not have enough free space. Come back when you have <tons> of free space.`

Reviewer: ```suggestion
	blocked `The cargo is ready to load, however, you do not have enough free space. Come back when you have <tons> of free space.`
```
alternatively
```suggestion
	blocked `The cargo is ready to load, but you do not have enough free space. Come back when you have <tons> of free space.`
```

Author: This should be fixed in Remnant Keystone 4 too then

				`	"The Korath?"`
					goto korath
				`	"The Hai?"`
				`	"The Pug?"`
					to display
						has "main plot completed"
					goto pug
				`	"The Sheragi?"`
				`	"The Predecessors?"`
					to display
						has "Successors: history"
					goto predecessors
			`	"A good guess. Both the Hai and the Sheragi would make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard. I heard there are great debates about the two species going on to this day, especially after the discoveries in Ssil Vida, but I am not the person to ask about them.`
				goto continue
			label pug
			`	"Interesting choice. We do know that they used to have a presence in this area at some point, but there is little use in speculating about their purposes with so little information, especially when there are more likely candidates. Either the Hai or the Sheragi could make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
				goto continue
			label predecessors

> 			`	He shakes his head with some disappointment. "Far more time has passed between the end of the Builders and the Successor's discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi could make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Reviewer: ```suggestion
			`	He shakes his head with some disappointment. "Far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi could make some sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
```

			`	"Interesting guess. We do know that they used to have a presence in this area at some point, but there is little use in speculating about their purposes with so little information, especially when there are more likely candidates. Either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
				goto continue
			label predecessors

> 			`	He shakes his head with some disappointment. "I may not know much on them, but I can tell you that far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`

Reviewer: ```suggestion
			`	He shakes his head with some disappointment. "I may not know much of them, but I can tell you that far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
```
or
```suggestion
			`	He shakes his head with some disappointment. "I may not know much about them, but I can tell you that far more time has passed between the end of the Builders and the Successors' discovery of spaceflight than it has between the latter and now. Keep it in mind, we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
```

			label builders
			`	The Builders seem to have found some interesting uses for it, however. Among the destroyed settlements all around Alix, vast mining facilities not only survived the attacks, but still see a constant stream of Ka'mar vehicles coming to gather their mineral cargo and deliver it to the imposing Ka'het station orbiting the same star.`
			`	Strangely enough, this moon does not show any sign of strip mining, or in fact any digging at all. After operating mining facilities for a hundred thousand years, the surface should show clearly visible signs of such an intensive operation, but there seem to be none.`
			choice
				`	(Fly closer to the mining outposts to investigate.)`
				`	(Move to the target the Remnant gave you.)`
					goto quarantine
			`	You pitch down your ship and steer toward the nearest outpost: a long series of short buildings with angled roofs, reminiscent of a solar field in your ship's screen. After coming closer, however, you realize your initial guess was wrong: these are not akin to roofs, but to sails, composed of a pale white textile. A thin layer of dust covers their surface, mostly gathered toward the bottom.`

> 			`	You try to come closer, but the compass on your ship suddenly starts spinning randomly! Alarmed, you swiftly reverse course to move away from the facility, and the instrumentation soon returns nominal again after a reboot. The automated outpost is clearly emitting a strong magnetic field, but the connection among all this isn't immediately obvious.`

Reviewer 3: ```suggestion
			`	You try to come closer, but the compass on your ship suddenly starts spinning randomly! Alarmed, you swiftly reverse course to move away from the facility, and the instrumentation soon returns to normal again after a reboot. The automated outpost is clearly emitting a strong magnetic field, but the connection among all this isn't immediately obvious.`
```

			choice
				`	(Do a close flyby, even if the compass goes haywire.)`
				`	(Try to search around the facility, keeping a safe distance.)`
					goto coward
				`	(Give up and move to the original target.)`
					goto quarantine
			`	A brief exploration doesn't turn up anything you hadn't already seen earlier, and all you can do is watch as the mesmerizing flow of Faes'mar in and out of the facility continues undisturbed below you.`
			label coward

> 			`	Just as you're starting to feel this mystery will remain unsolved, you have a stroke of luck: a small meteorite hits the surface a few kilometers away from you, throwing a cloud of debris in every direction. You check the composition of the nearly imperceptible atmosphere near the surface, and find it composed almost entirely of precious metal particles.`

Reviewer 3: ```suggestion
			`	Just as you're starting to feel that this mystery will remain unsolved, you have a stroke of luck: a small meteorite hits the surface a few kilometers away from you, throwing a cloud of debris in every direction. You check the composition of the nearly imperceptible atmosphere near the surface, and find it composed almost entirely of precious metal particles.`
```

			`	The overall picture has become clearer: the frequent asteroid impacts on a moon like this raise enough meteorite dust to effectively shroud the planet in an extremely sparse atmosphere, which the Builder machines attract and gather so that they can be transported off-planet. It's a slow and cumbersome process, but it's also one that was able to work for tens of thousands of years without maintenance.`
			label quarantine
			`	You record a brief log of your thoughts, and move on to your actual destination. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, the map eventually signals you have reached your destination: a plain, unremarkable plateau extending for kilometers in every direction.`
			`	You pitch down and lower your altitude to look for debris, but you're not able to find anything useful. There were many ruins on the way here that could have certainly been a quarantine site at some point, but there's absolutely nothing here, as if it was never even inhabited in the first place.`

> 			`	Perplexed, you turn your ship around and activate the high-power scanners the Remnant gave you. The task is very tedious: you have to pass over every bit of terrain going just at the right speed, careful not to skew the data with the wrong movement, to allow the instruments to function correctly.`

Reviewer 3: ```suggestion
			`	Perplexed, you turn your ship around and activate the high-power scanners the Remnant gave you. The task is very tedious: you have to pass over every bit of terrain going at just the right speed, careful not to skew the data with any abrupt movements, so that the instruments can function correctly.`
```

			`	It takes an hour, but a familiar beep from the cockpit panel finally signals you can come back to <destination> to deliver the data.`
	on complete
		event "ka'het: quarantine site" 25
		conversation

> 			`Unlike your last visit, the testing range on <planet> has returned to an eerily empty condition once again. Gone are the massive fleets occupying every parking spot, and the few ships that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`

Reviewer 3: ```suggestion
			`Unlike your last visit, the testing range on <planet> has become eerily empty once again. Gone are the massive fleets occupying every parking spot, and the few ships that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`
```

			`	With a datapad in your hand and the pad workers unloading your used mission equipment, you start looking around for someone to deliver the data to. The only people you can find nearby are the ones working on your ship, as well as the pad manager you had met earlier.`
			`	"They relented!" she signs excitedly after noticing you. "It took a few days, but they finally accepted that we do not have the facilities nor the personnel to handle such traffic, and moved the satellite to a more appropriate laboratory. Somewhere on Aventine, I believe."`
			choice
				`	"Is there someone I can give this data to?"`
				`	"The location you gave me did not seem to have been inhabited."`

> 			`	You give her a brief summary of your mission, and of your findings. "I am afraid the people you were supposed to deliver this data to have left together with the satellite," she replies, perplexed. "It has been some busy days for all of us recently, and some steps were skipped to accelerate the process.`

Reviewer 3: ```suggestion
			`	You give her a brief summary of your mission, and of your findings. "I am afraid the people you were supposed to deliver this data to have left together with the satellite," she replies, perplexed. "It has been busy for all of us recently, and some steps were skipped to accelerate the process."`
```

> 			`	You give her a brief summary of your mission, and of your findings. "I am afraid the people you were supposed to deliver this data to have left together with the satellite," she replies, perplexed. "It has been some busy days for all of us recently, and some steps were skipped to accelerate the process.`

Reviewer 3: What does she mean by `accelerate the process` here?

				`	(Try to search around the facility, keeping a safe distance.)`
					goto coward
				`	(Give up and move to the original target.)`
					goto quarantine
			`	A brief exploration doesn't turn up anything you hadn't already seen earlier, and all you can do is watch as the mesmerizing flow of Faes'mar in and out of the facility continues undisturbed below you.`
			label coward
			`	Just as you're starting to feel that this mystery will remain unsolved, you have a stroke of luck: a small meteorite hits the surface a few kilometers away from you, throwing a cloud of debris in every direction. You check the composition of the nearly imperceptible atmosphere near the surface, and find it composed almost entirely of precious metal particles.`
			`	The overall picture has become clearer: the frequent asteroid impacts on a moon like this raise enough meteorite dust to effectively shroud the planet in an extremely sparse atmosphere, which the Builder machines attract and gather so that they can be transported off-planet afterwards. It's a slow and cumbersome process, but it's also one that was able to work for tens of thousands of years without maintenance.`
			label quarantine

> 			`	You record a brief log of your thoughts, and move on to the target location. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, you eventually reach your destination, unremarkable plateau extending for kilometers in every direction.`

Reviewer: ```suggestion
			`	You record a brief log of your thoughts, and move on to the target location. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, you eventually reach your destination: an unremarkable plateau extending for kilometers in every direction.`
```

			label continue
			`	"Whatever happened, the scars left by this short encounter were not merely psychological, as this satellite shows. Some kind of virus started spreading soon after the first contact, possibly of alien origin. We do not know the impact it had on their civilization, but it must have been quite severe: Alix appears to have been used as a quarantine facility at the time, and satellites like this had the sole function of warning incoming ships of the danger.`
			`	"If you find anything else, do not hesitate to tell us. The analysis is still ongoing, but I have been able to spare 300,000 credits to deposit on your account to show our gratitude."`
			choice
				`	"You can keep the money. I would rather continue exploring further."`
					goto keep
				`	"Who are the Sheragi?"`
					to display
						not "event: rim archaeology results"
				`	"Thank you."`
					goto capitalist

> 			`	"You might not have heard the name, but you have certainly been into their territory. They are a long extinct species, once inhabiting what has become the Rim and the South of human space. The same area where, not too long ago, you took the material we are currently using to power Ssil Vida," he replies, making a gesture in the direction of the station's reactor.`

Reviewer: It's been a while since I read the missions, so I could be wrong, but didn't the player get the fuel from Kornephoros instead of taking it there?
```suggestion
			`	"You might not have heard the name, but you have certainly been into their territory. They are a long extinct species, once inhabiting what has become the Rim and the South of human space. The same area where, not too long ago, you got the material we are currently using to power Ssil Vida," he replies, making a gesture in the direction of the station's reactor.`
```

			choice
				`	(Ask her about your mission.)`
				`	"What are they doing with my ship?"`
			`	"It seems there must have been some miscommunications along the way," she signs in reply to your queries. "Sorry, but we have not had anything close to this amount of traffic for at least the past thirty years. We cannot afford to waste even a single minute right now.`
			`	"Ever since someone brought us that satellite a few days ago," she continues, seemingly oblivious about who you are, "there have been a constant stream of captains wanting to take a look at it and do all kinds of analysis. We are not staffed for this kind of traffic, not even close."`
			`	You try to inquire further about your mission, but she does not seem to know any more than you do. She redirects you again to the pad, pointing toward the bill of landing attached to the cargo they are mounting on your ship.`
			`	At last, you find in there the information you needed: your payload is a large penetrating scanner, and your task is to use it just above the surface of <stopovers> in a location near the south pole of that moon to try to locate the quarantine facility.`
				accept
	npc
		government "Remnant"

> 		personality heroic launching lingering

Reviewer: I must not have seen your comment on the original suggestion about `uninterested`, but yeah, that makes sense
```suggestion
		personality heroic launching lingering uninterested
```

	on complete
		event "ka'het: quarantine site" 25
		conversation
			`Unlike your last visit, the testing range on <planet> has become eerily empty once again. Gone are the massive fleets occupying every parking spot, and the few ships that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`
			`	With a datapad in your hand and the pad workers unloading your used mission equipment, you start looking around for someone to deliver the data to. The only people you can find nearby are the ones working on your ship, as well as the pad manager you had met earlier.`
			`	"They relented!" she signs excitedly after noticing you. "It took a few days, but they finally accepted that we do not have the facilities nor the personnel to handle such traffic, and moved the satellite to a more appropriate laboratory. Somewhere on Aventine, I believe."`
			choice
				`	"Is there someone I can give this data to?"`
				`	"The location you gave me did not seem to have been inhabited."`

> 			`	You give her a brief summary of your mission, and of your findings. "I am afraid the people you were supposed to deliver this data to have left together with the satellite," she replies, perplexed. "It has been busy for all of us recently, and some steps were skipped to accelerate the process."`

Reviewer: ```suggestion
			`	You give her a brief summary of your mission, and of your findings. "I am afraid the people you were supposed to deliver this data to have left together with the satellite," she replies, perplexed. "It has been busy for all of us recently, and some steps were skipped to accelerate the process.`
```

			`	"If you leave the datapad here, I promise I will make sure it gets into the right hands. I am sure they will contact you themselves after things have calmed down a bit." You hand her the tablet with the info you recorded, hopeful it will reach its destination, and prepare to leave once again.`

event "ka'het: quarantine site"

event "ka'het: alix discoveries"
	planet Alix

> 		add description `	Long before becoming an automated mining outpost, this moon was originally designated as a Builder quarantine site.`

Reviewer: ```suggestion
		add description `	Long before becoming an automated mining outpost, this moon was designated as a Builder quarantine site.`
```
Presumably if it was "long before", we already know that it was "originally".

					goto signal
			`	"I have a few ideas, but there is no way to be sure. One of the most frequent causes is the software successfully rebooting after a memory failure, practically resetting it to the factory setup, but without more information this is at best a guess on my part."`
			label signal
			`	You hand him a data chip with the communication you received from the satellite, which he quickly puts into his pocket. "I will try to take a few hours off to give it a first pass with the instruments we have here. While they are not the best suited for this sort of analysis, I remember a few tricks that might get something usable out of it.`
			`	"In the meantime, feel free to take a look at the changes we have been making here. I will give you a call from the spaceport when I have something to show."`

mission "Ka'het: Voice From The Past 1B"
	invisible
	source "Ssil Vida"
	to offer
		has "Ka'het: Voice From The Past 0: done"
		not "Ka'het: Voice From The Past 1: offered"
	on offer
		conversation
			`Do you want to ask the Remnant about the communication received from the Builder satellite?`
			choice
				`	(Yes.)`
				`	(Not now.)`
					defer

> 			`With the ongoing construction preventing you from reaching most of the station, you try to look around near the landing pad. It takes almost half an hour before you give up; there's too many people to try to find someone who knows you, and everyone else seems far too busy to stop and listen to your story. Perhaps you will have better luck in the spaceport of a major planet.`

Author: Added

mission "Ka'het: Voice From The Past 2"
	name "Outpost on Kumkapi"
	description "Bring Gavriil to <destination> to prepare for the capture of the Builder satellite."
	blocked `This mission requires one empty bunk to carry Gavriil with you.`
	source "Ssil Vida"
	destination "Kumkapi"
	passengers 1
	deadline 8
	to offer
		has "Ka'het: Voice From The Past 1: done"
	on offer

> 		fail "Ka'het: Voice From The Past 0"

Author: Done

			`	You help them load the satellite capture equipment in your ship, and make sure to leave enough space for the spacecraft itself. "I will stay here to help them prepare for the satellite," Gavriil signs, "and hopefully remain updated with Ssil Vida on the progress made in my absence. See you in a month or two."`
				accept
	npc assist
		government "Builder"
		system "Fereti"
		personality derelict target waiting
		ship "Builder Satellite" "Valii'sei"
		conversation
			`You approach the spacecraft, and activate the equipment the Remnant gave you. Slowly, a large robotic claw, padded with a thick gray insulation, emerges out of your cargo hold. It takes a few tries, but you're finally able to put the satellite into your cargo bay and get the fittings to keep it in place prepared.`
				launch

mission "Ka'het: Voice From The Past 4"
	landing
	name "Return Gavriil"
	description "Bring Gavriil back to <destination>."
	source "Kumkapi"
	destination "Ssil Vida"

> 	deadline 8

Reviewer: Presumably, Gavriil is still on leave and needs to get back soon.

> 	deadline 8

Author: Yeah, but you can take as much time as you want to catch the satellite so it feels a bit random. That was the thinking on stream, at least

				goto continue
			label predecessors
			`	"Those that came before?" He shakes his head with some disappointment. "I may not know much about them, but I can tell you that far more time has passed between the end of the Builders and the first Predecessor artifacts than it has between the latter and now. Keep it in mind that we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
				goto continue
			label korath
			`	"Not this time," he signs, amused. "More time has passed between the end of the Builders and the Korath's discovery of spaceflight than it has between the latter and now. Keep it in mind that we are talking about something that happened perhaps a hundred thousand years ago. In this context, either the Hai or the Sheragi would make more sense - the timeline fits well enough, they were known to be quite belligerent, and their potential reach was large enough to include the Graveyard.`
			label continue

> 			`	"Whatever happened, the scars left by this short encounter were not merely psychological, as this satellite shows. Some kind of virus started spreading soon after the first contact, possibly of alien origin. We do not know the impact it had on their civilization, but it must have been quite severe: Alix appears to have been used as a quarantine facility at the time, and satellites like this had the sole function of warning incoming ships of the danger.`

Author: Changed to `	"Whatever happened, the scars left by this short encounter were not merely psychological. What this satellite shows is that some kind of virus or bacteria started spreading soon after the first contact, possibly of alien origin. We do not know the impact it had on their civilization, but it must have been quite severe: Alix appears to have been used as a quarantine facility at the time, and satellites like this had the sole function of warning incoming ships of the danger.`
Not sure if that's enough

			`	"If you find anything else, do not hesitate to tell us. The analysis is still ongoing, but I have been able to spare 300,000 credits to deposit on your account to show our gratitude."`
			choice
				`	"You can keep the money. I would rather continue exploring further."`
					goto keep
				`	"Who are the Sheragi?"`
					to display
						not "event: rim archaeology results"
				`	"Thank you."`
					goto capitalist
			`	"You might not have heard the name, but you have certainly been into their territory. They are a long extinct species, once inhabiting what has become the Rim and the South of human space. The same area where, not too long ago, you got the material we are currently using to power Ssil Vida," he replies, making a gesture in the direction of the station's reactor.`

> 			`	"I can tell you that, as far as we know, they were roughly concurrent with the Builders; but this is the wealth of information I learnt during the work in Postverta. It is a continuously evolving field, and I have not been able to follow it closely beyond the engineering research on this station."`

Author: Added 
			action
				set "remnant: sheragi question"

				`	(Try to search around the facility, keeping a safe distance.)`
					goto coward
				`	(Give up and move to the original target.)`
					goto quarantine
			`	A brief exploration doesn't turn up anything you hadn't already seen earlier, and all you can do is watch as the mesmerizing flow of Faes'mar in and out of the facility continues undisturbed below you.`
			label coward
			`	Just as you're starting to feel that this mystery will remain unsolved, you have a stroke of luck: a small meteorite hits the surface a few kilometers away from you, throwing a cloud of debris in every direction. You check the composition of the nearly imperceptible atmosphere near the surface, and find it composed almost entirely of precious metal particles.`
			`	The overall picture has become clearer: the frequent asteroid impacts on a moon like this raise enough meteorite dust to effectively shroud the planet in an extremely sparse atmosphere, which the Builder machines attract and gather so that they can be transported off-planet. It's a slow and cumbersome process, but it's also one that was able to work for tens of thousands of years without maintenance.`
			label quarantine

> 			`	You record a brief log of your thoughts, and move on to your actual destination. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, the map eventually signals you have reached your destination: a plain, unremarkable plateau extending for kilometers in every direction.`

Reviewer 4: ```suggestion
			`	You record a brief log of your thoughts, and move on to the target location. The mining facilities you see become progressively smaller and more sporadic, until they disappear completely. Flying just above a series of increasingly featureless highlands, you eventually reach your destination, unremarkable plateau extending for kilometers in every direction.`
```
It sounds better to keep this more generalized than to specifically say my map is what's telling me I'm at the location.

			`	You pitch down and lower your altitude to look for debris, but you're not able to find anything useful. There were many ruins on the way here that could have certainly been a quarantine site at some point, but there's absolutely nothing here, as if it was never even inhabited in the first place.`
			`	Perplexed, you turn your ship around and activate the high-power scanners the Remnant gave you. The task is very tedious: you have to pass over every bit of terrain going at just the right speed, careful not to skew the data with any abrupt movements, so that the instruments can function correctly.`
			`	It takes an hour, but a familiar beep from the cockpit panel finally signals you can come back to <destination> to deliver the data.`
	on complete
		event "ka'het: quarantine site" 25
		conversation

> 			`Unlike your last visit, the testing range on <planet> has become eerily empty once again. Gone are the massive fleets occupying every parking spot, and the few ships that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`

Author: I have projected my hatred of large crowds on first last. Is enticed better?

> 			`Unlike your last visit, the testing range on <planet> has become eerily empty once again. Gone are the massive fleets occupying every parking spot, and the few ships that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`

Reviewer 4: I'm not sure, because there's no explicit reason mentioned why the player would be enticed. It says the landing pads are eerile quiet, which, without the context of "this has been hectic and annoying", seems like a really random, specific emotion to feel. I think either expanding a tiny bit about it finally not being hectic/busy would be better instead, or just remove the emotion period.

> 			`Unlike your last visit, the testing range on <planet> has become eerily empty once again. Gone are the massive fleets occupying every parking spot, and the few ships that are still here are rarely bigger than a Petrel. Heartened, you set sight on a random pad in the vicinity of the facility and touch down softly.`

Author: `Unlike your last visit, the testing range on <planet> has become eerily empty once again. Gone are hectic crowds and the massive fleets occupying every parking spot: the few ships that are still here are rarely bigger than a Petrel. Enticed, you set sight on a random pad in the vicinity of the facility and touch down softly.` ?
